HeliPort/Appearance/StatusMenu.swift

```diff
@@ -326,9 +361,63 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
         }
 
         DispatchQueue.global(qos: .background).async {
-            var info = station_info_t()
-            get_station_info(&info)
-            Log.debug(String(format: "current rate=%03d", info.rate))
+            var disconnectName = NSLocalizedString("Unavailable", comment: "")
+            var ipAddr = NSLocalizedString("Unavailable", comment: "")
+            var routerAddr = NSLocalizedString("Unavailable", comment: "")
+            var internet = NSLocalizedString("Unavailable", comment: "")
+            var security = NSLocalizedString("Unavailable", comment: "")
+            var bssid = NSLocalizedString("Unavailable", comment: "")
+            var channel = NSLocalizedString("Unavailable", comment: "")
+            var countryCode = NSLocalizedString("Unavailable", comment: "")
+            var rssi = NSLocalizedString("Unavailable", comment: "")
+            var noise = NSLocalizedString("Unavailable", comment: "")
+            var txRate = NSLocalizedString("Unavailable", comment: "")
+            var phyMode = NSLocalizedString("Unavailable", comment: "")
+            var mcsIndex = NSLocalizedString("Unavailable", comment: "")
+            var staInfo = station_info_t()
+            Log.debug(String(format: "current rate=%03d", staInfo.rate))
+            if self.status == ITL80211_S_RUN && get_station_info(&staInfo) == KERN_SUCCESS {
+                let macAddress = String(self.macItem.title)
+                    .replacingOccurrences(of: "Address: ", with: "", options: .regularExpression, range: nil)
+                let ipInfo = NetworkManager.getCurrentIpAddress(matchingMAC: macAddress)
+                disconnectName = String(cString: &staInfo.ssid.0)
+                ipAddr = ipInfo.0
+                routerAddr = ipInfo.1
+                internet = "Reachable"
+                security = "Unknown"
+                bssid = String(format: "%02x:%02x:%02x:%02x:%02x:%02x",
+                               staInfo.bssid.0,
+                               staInfo.bssid.1,
+                               staInfo.bssid.2,
+                               staInfo.bssid.3,
+                               staInfo.bssid.4,
+                               staInfo.bssid.5
+                )
+                channel = String(staInfo.channel) + " (" +
+                    (staInfo.channel <= 14 ? "2.4 GHz" : "5 GHz") + ", " +
+                    String(NetworkManager.channelToMHz(chan: staInfo.channel, is5g: staInfo.channel > 14)) + " MHz)"
```

Comment:
The last figure here is intended to be the channel width, for example:
```
Channel: 11 (2.4 GHz, 20 MHz)
```

Reply:
Yup I figured out that the MHz I was converting to was for channel, need to figure out how to calculate channel width.

Reply:
I added a placeholder in the meantime depending on the op_mode, however that has not been implemented either.